internal/pkg/show/config/plugins/zentao.yaml

```diff
@@ -0,0 +1,51 @@
+tools:
+  # name of the tool
+  - name: zentao
+    # id of the tool instance
+    instanceID: default
+    # format: name.instanceID; If specified, dtm will make sure the dependency is applied first before handling this tool
+    dependsOn: []
+    # options for the plugin
+    options:
+      # namespace for zentao application
+      namespace: 'zentao'
+      # storageClassName used to match pv and pvc
+      storageClassName: 'zentao-storage'
+      # two PersistentVolumes for zentao and mysql should be specified
+      persistentVolume:
+        # name of zentao pv
+        zentaoPVName: 'zentao-pv'
+        # capacity of zentao pv
+        zentaoPVCapacity: '1G'
+        # name of mysql pv
+        mysqlPVName: 'mysql-pv'
+        # capacity of mysql pv
+        mysqlPVCapacity: '1G'
+      # two PersistentVolumeClaims for zentao and mysql should be specified
+      persistentVolumeClaim:
+        # name of zentao pvc
+        zentaoPVCName: 'zentao-pvc'
+        # capacity of zentao pvc
+        zentaoPVCCapacity: '1G'
+        # name of mysql pvc
+        mysqlPVCName: 'mysql-pv'
+        # capacity of mysql pvc
+        mysqlPVCCapacity: '1G'
+      # zentao application is deployed by K8S Deployment
+      deployment:
+        # name of zentao deployment
+        name: 'zentao-dp'
+        # number of application replica
+        replicas: 3
+        # zentao image
+        image: 'easysoft/zentao:latest'
+        # initial password name for mysql database, you can specify any name you like
+        mysqlPasswdName: 'MYSQL_ROOT_PASSWORD'
+        # initial password value for mysql database, you can specify any value you like
+        mysqlPasswdValue: '1234567'
+      # zentao application is exposed via K8S Service
+      service:
+        # name of zentao service
+        name: 'zentao-svc'
+        # nodePort of zentao service, currently zentao plugin only support `nodePort` type
+        nodePort: 30081
```

Comment:
No newline at the end of the file zentao.yaml. Please check it.

Reply:
@hxcGit 

> No newline at the end of the file zentao.yaml. Please check it.

Reply:
will fix it